### PR TITLE
Crl/add clover

### PR DESCRIPTION
**Pull request by ChloeL19**
Added the following script files and submodules:
Clover/ --> a submodule created from this repo: https://github.com/ChuyueSun/Clover/tree/main 
clover_loader.py --> data loader for the Clover dataset
clover_config.py --> configuration settings for loading train/test portions of the Clover dataset
run_clover.py --> run VMCTS on the Clover benchmark
gen_triples.py --> create triples for DPO based on the Clover benchmark

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,17 +0,0 @@
-accelerate
```
Comment (namin) [inline requirements.txt:1]: why did you delete this file?

Comment (ChloeL19) [reply requirements.txt:1]: that was also a mistake

File: run_clover.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+from montecarlo.node import Node
```
Comment (namin) [inline run_clover.py:1]: run_clover shouldn't duplicate the run, instead, it should be parametric in the run like experiments, maybe?

Comment (ChloeL19) [reply run_clover.py:1]: Going to tackle in separate PR
